doc/Language/faq.pod6

```diff
@@ -299,6 +299,77 @@ dependencies within a single compilation unit (e.g., file) are possible
 through stubbing. Therefore another possible solution is to move
 classes into the same compilation unit.
 
+
+=head1 Common operations
+
+=head2 String: How can I parse and get a L<number|/language/numerics> from a L<string|/language/type/Str>
+
+Use the L<+ prefix|/language/operators#prefix_+>:
```

Comment:
This is an example of contextualizing; it's mentioned explicitly in the document on contexts. It's OK to add new examples, but referring to the existing content is even better.

Reply:
200% agree. All that is an excuse to link to the doc. I did not know this page.